cloudhub-2/modules/ROOT/pages/ch2-configure-horizontal-autoscaling.adoc

```diff
@@ -0,0 +1,138 @@
+= Configuring Horizontal Autoscaling (HPA) for CloudHub 2.0 Deployments
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+[IMPORTANT]
+This feature applies only to select customers in the US control plane who are opted in to the new pricing and packaging model. For more details, visit xref:general::pricing.adoc[]
+
+You can configure CPU-based horizontal scaling for Mule applications to make them responsive to CPU usage by automatically scaling up or down the deployment replicas as needed.
+
+In Kubernetes, a Horizontal Pod Autoscaler (HPA) automatically updates a workload resource, with the aim of automatically scaling the workload to match demand. Horizontal scaling means that the response to increased load is to deploy more pods. For more information, visit the https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale[Kubernetes documentaton^]. 
+
+== Configure Horizontal Pod Autoscaling
+
+To configure horizontal autoscaling for Mule apps deployed to CloudHub 2.0, follow these steps:
+
+. From Anypoint Platform, select *Runtime Manager* > *Applications*.
+. Click *Deploy application*.
+. In the *Runtime* tab, check the *Enable Autoscaling* box.
+. Set the minimum and maximum *Replica Count* limits.
```

Comment:
Can we add a step after this to :
Choose Replica Size. Integration-Advanced customers can choose from Micro, Micro.Mem and Small replica sizes.  Customers on other tiers can choose from Micro, Micro.Mem replica sizes. [Link to CloudHub2.0 replica size options in the new P&P: https://docs.mulesoft.com/general/pricing#cloudhub-2-0-application-deployment]